### ISSUE TITLE
Regression in purchase/invoice.py with MULTICURRENCY

### ISSUE THREAD

**Issue by blaggacao** [odoo/odoo#9252]
Commit: 0ea31c272c1e01a2f064231ef9ed82e3b3c2dd39
 
```
  File "/home/src/devco9/addons/purchase/invoice.py", line 116, in _anglo_saxon_purchase_move_lines
    valuation_price_unit = company_currency.compute(inv.currency_id, valuation_price_unit, context={'date': inv.date_invoice})
  File "/home/src/devco9/openerp/api.py", line 221, in wrapper
    return new_api(self, *args, **kwargs)
ValueError: "compute() got an unexpected keyword argument 'context'" while evaluating
u'action_move_create()'
```

```
          if inv.currency_id.id != company_currency.id:
                            valuation_price_unit = company_currency.compute(inv.currency_id, valuation_price_unit, context={'date': inv.date_invoice})
```

**Comment by blaggacao** [comment]
@nim-odoo 
You might have a look: https://github.com/odoo/odoo/blame/64a6a84073659ace4f43650b05ea943c42a97b4f/addons/purchase/invoice.py#L115-L116

At the moment, I don't know what is to be achieved by this context.
I'm happy to help with further analysis, if you drop me some hints.

**Comment by qdp-odoo** [comment]
@nim-odoo : sorry Nicolas, i did not see you self-assigned this issue... 

it has been fixed in 098fc9f

**Comment by blaggacao** [comment]
WoW!!! :smile:
